mrmustard/lab_dev/transformations/base.py

```diff
@@ -184,9 +194,13 @@ def from_symplectic(
             name=name,
         )
 
+    def inverse(self) -> Transformation:
```

Comment:
```suggestion
    def inverse(self) -> Unitary:
```

Reply:
Changed to `DualView` as this is the return type of `self.dual`